README.md

```diff
@@ -12,28 +12,28 @@
         - [Unit Test](#unit-test)
           - [Mock the ledger object](#mock-the-ledger-object)
           - [Assert the result of execution](#assert-the-result-of-execution)
-        - [Use Scalar DL Emulator](#Use-Scalar-DL-Emulator)
+        - [Use ScalarDL Emulator](#Use-Scalar-DL-Emulator)
     - [Develop client (application)](#Develop-client-application)
       - [Register certificate](#register-certificate)
       - [Register a contract](#register-a-contract)
       - [Execute a contract](#Execute-a-contract)
       - [Validate ledger data integrity](#Validate-ledger-data-integrity)
 
-In this tutorial we introduce how to implement a simple asset management application on Scalar DLT solution.
+In this tutorial we introduce how to implement a simple asset management application on ScalarDL solution.
 If you haven't tried this application yet, please refer to [here](./manual/README.md).
 
 ## Overview
 
-Scalar DLT is a distributed ledger platform.
-A Scalar DL network exposes a key-value semantics called ledger to clients.
+ScalarDL is a distributed ledger platform.
```

Comment:
Can you update the description to the current one? (I think we need a consistent description)